silence/silence.go

```diff
@@ -144,3 +162,25 @@ func Remove(silenceID string) error {
 	}
 	return fmt.Errorf("there was an error unsilencing the cluster")
 }
+
+// ExpiresIn returns bool if the remaining time on the AlertManager Silence is less than the expiryPeriod
```

Comment:
This comment's functionName is wrong and needs to be updated as well.

Reply:
Fixed.